src/components/workers-page/workers-tab/workers-tab.component.tsx

```diff
@@ -3,20 +3,53 @@ import Table from "@material-ui/core/Table";
 import TableBody from "@material-ui/core/TableBody";
 import TableCell from "@material-ui/core/TableCell";
 import TableContainer from "@material-ui/core/TableContainer";
-import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
-import { WorkerType, WorkerTypeHelper } from "../../../common-models/worker-info.model";
+import { WorkerData, WorkerTypeHelper } from "../../../common-models/worker-info.model";
 import { useSelector } from "react-redux";
 import { ApplicationStateModel } from "../../../state/models/application-state.model";
 import { Button } from "../../common-components";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import { EnhancedTableHeaderComponent } from "./enhanced-table-header-component";
+import { ArrayHelper, Comparator, Order } from "../../../helpers/array.helper";
+import { StringHelper } from "../../../helpers/string.helper";
+import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
 
-interface WorkerData {
-  name: string;
-  type: WorkerType;
-  time: number;
+function descendingComparator<T>(a: T, b: T, orderBy: keyof T): Comparator {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
 }
 
+function getComparator<Key extends number | string | symbol>(
+  order: Order,
+  orderBy: Key
+): (a: { [key in Key]: number | string }, b: { [key in Key]: number | string }) => Comparator {
+  return order === "desc"
+    ? (a, b): Comparator => descendingComparator(a, b, orderBy)
+    : (a, b): Comparator => -descendingComparator(a, b, orderBy);
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      paddingTop: 0,
+      width: "100%",
+    },
+    table: {
+      borderTop: "2px solid #D5E6FE",
```

Comment:
kolor przydało by się ze zmiennych brać, chyba, że będzie używany tylko w tym miejscu

Reply:
A to zapomniałem wyrzucić. Poprawię :)